exonum/src/sandbox/consensus/mod.rs

```diff
@@ -32,11 +32,8 @@ mod unsynchronized_message;
 // LOCK
 
 // - Send precommit when get lock   //covered in lock_to_propose_when_get_2_3_prevote_positive()
-// - if haven’t incompatible prevotes
+// - if haven't incompatible prevotes
```

Comment:
Maybe "If there are no incompatible prevotes"?

Reply:
Dunno. Haven't read it yet.